packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -590,6 +590,144 @@ void main() {
+        ThemeData().sliderTheme.copyWith(
+          overlayShape: SliderComponentShape.noOverlay,
+          thumbShape: SliderComponentShape.noThumb,
+          showValueIndicator: ShowValueIndicator.never,
+        ),
+        value: 0.5,
+        divisions: 4
+    ));
+
+    // 2 track segments.
+    expect(sliderBox, paintsExactlyCountTimes(#drawRect, 2));
+    // 5 tick marks.
+    expect(sliderBox, paintsExactlyCountTimes(#drawCircle, 5));
+
+    // Pump a slider with a track, tick marks, and a thumb.
+    await tester.pumpWidget(_buildApp(
+      ThemeData().sliderTheme.copyWith(
+        overlayShape: SliderComponentShape.noOverlay,
+        showValueIndicator: ShowValueIndicator.never,
+      ),
+      value: 0.5,
+      divisions: 4
+    ));
+
+    // 2 track segments.
+    expect(sliderBox, paintsExactlyCountTimes(#drawRect, 2));
+    // 5 tick marks and 1 thumb.
+    expect(sliderBox, paintsExactlyCountTimes(#drawCircle, 6));
+
+    // Tap the center of the track and wait for animations to finish.
+    final Offset center = tester.getCenter(find.byType(Slider));
+    TestGesture gesture = await tester.startGesture(center);
+    await tester.pumpAndSettle();
+
+    // Tapping the center would usually produce an overlay and a value
+    // indicator. However, these shapes are currently removed, so no extra
+    // paint shapes are expected, and there are still the same number of
+    // shapes on the screen.
+
+    // 2 track segments.
+    expect(sliderBox, paintsExactlyCountTimes(#drawRect, 2));
+    // 5 tick marks and 1 thumb.
+    expect(sliderBox, paintsExactlyCountTimes(#drawCircle, 6));
+
+    await gesture.up();
+    await tester.pumpAndSettle();
+
+    // Pump a slider with a track, tick marks, a thumb, and an overlay.
+    await tester.pumpWidget(_buildApp(
+      ThemeData().sliderTheme.copyWith(
+        showValueIndicator: ShowValueIndicator.never,
+      ),
+      value: 0.5,
+      divisions: 4
+    ));
+
+    gesture = await tester.startGesture(center);
+    await tester.pumpAndSettle();
+
+    // Now that there is a tap, and the overlay is not the empty shape,
+    // an extra circle is expected to be painted.
+
+    // 2 track segments.
+    expect(sliderBox, paintsExactlyCountTimes(#drawRect, 2));
+    // 5 tick marks, 1 thumb, and 1 overlay.
+    expect(sliderBox, paintsExactlyCountTimes(#drawCircle, 7));
+
+    await gesture.up();
+    await tester.pumpAndSettle();
+
+    // Pump a slider with a track, tick marks, a thumb, an overlay, and a
+    // value indicator.
+    await tester.pumpWidget(_buildApp(
+        ThemeData().sliderTheme.copyWith(
+          showValueIndicator: ShowValueIndicator.always,
+        ),
+        value: 0.5,
+        divisions: 4
+    ));
+
+    gesture = await tester.startGesture(center);
+    await tester.pumpAndSettle();
+
+    // Now that there is a tap, and the value indicator is specified to be
+    // , shown an path is expected to be painted.
```

Comment:
Not totally sure what this supposed to say, but I think there is a typo on the second line.

Reply:
was a typo, updated